client/components/ScenariosList/index.jsx

```diff
@@ -23,10 +25,11 @@ class ScenariosList extends Component {
     }
 
     render() {
-        const category = this.props.location.pathname.slice(1);
-        let scenarios =
-            (this.props.scenarios &&
-                this.props.scenarios.filter(({ categories }) => {
+        const { category, scenarios } = this.state;
```

Comment:
Why are we switching to the component's state here instead of using the scenarios that are in props? I feel like with your work to get them into Redux, the scenarios that are props will be the most up-to-date data. Let me know if I have that wrong though.

Reply:
Oh, it's possible that you're totally right, let me check!

Reply:
Fixed! Sorry about that :¯\_(ツ)_/¯